pyActigraphy/io/bba/bba.py

```diff
@@ -4,8 +4,39 @@
 import re
 
 from ..base import BaseRaw
-from accelerometer.utils import date_parser
-from accelerometer.summarisation import imputeMissing
+from pandas.tseries.frequencies import to_offset
+#from accelerometer.utils import date_parser
+#from accelerometer.summarisation import imputeMissing
+
+# Native implementation of date_parser and imputeMissing
+def date_parser(t):
+    """
+    Adapted from the accelerometer package (utils.py)
```

Comment:
Could you:
- provide a link to the accelerometer package? And the version you used (for future ref)? A tag is fine (for ex: v7.2.1);
- add a description for the function as well as for its input parameters (cf other functions in this class).

Reply:
I've provided a link to [accelerometer 7.2.1](https://pypi.org/project/accelerometer/) and the copyright notice as per the [license.md](https://github.com/OxWearables/biobankAccelerometerAnalysis/blob/ed0caad712a9686deb2bf3a37273c1dedd14beb6/LICENSE.md). I think our use is covered by paragraph 1.2, that grants licence to "reproduce, modify, transmit (...) for academic, research or other scholarly use.".